sklearn/impute/tests/test_impute.py

```diff
@@ -1500,3 +1498,44 @@ def test_most_frequent(expected, array, dtype, extra_value, n_repeat):
     assert expected == _most_frequent(
         np.array(array, dtype=dtype), extra_value, n_repeat
     )
+
+
+def test_impute_one_feature():
+    import numpy as np
+
+    from sklearn.datasets import fetch_california_housing
+    from sklearn.cross_decomposition import PLSRegression
+    from sklearn.experimental import enable_iterative_imputer  # noqa
+    from sklearn.impute import IterativeImputer
```

Comment:
You should move those imports at the top of the file. Probably, some are already there. 🙂